### ISSUE TITLE
Idea: Ability to allow moderators to communicate with each other

### ISSUE THREAD

**Issue by ThisIsMissEm** [swicg/activitypub-trust-and-safety#6]
This likely needs a group actor for moderators, and to have that advertised somewhere. Currently the "instance actor" is used for federating Flag activities, however this also isn't really defined anywhere.

**Comment by ThisIsMissEm** [comment]
There is this FEP that aims to help with discovery of "instance actors" (named in the FEP as "application actors") — https://codeberg.org/fediverse/fep/src/branch/main/fep/2677/fep-2677.md

**Comment by jfinkhaeuser** [comment]
It would likely be best to extend this to multiple possible instance related actors, and define a simple vocabulary for their purpose. E.g. the "instance actor", a group actor for moderators, a [securitytxt.org](https://securitytxt.org)-style actor, etc.

**Comment by ThisIsMissEm** [comment]
Yeah, but we need a way to advertise and discover all those actors, formalising the instance actor is the first step, since it's already in use; next step is defining additional actors for moderation, security, etc.

**Comment by jfinkhaeuser** [comment]
Well, I'm not saying you/this group is falling into that trap. But historically speaking across software engineering, it's easy to start with a specific idea, and when a second specific idea with staggering similarities comes along, you can either duct-tape it on, or make breaking changes. I'd like to avoid either with that comment, that's all :)
